src/api/models.py

```diff
@@ -1,19 +1,80 @@
 from flask_sqlalchemy import SQLAlchemy
+import os
+import sys
+from sqlalchemy import Column, ForeignKey, Integer, String, Table
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import relationship
+from sqlalchemy import create_engine
+
 
 db = SQLAlchemy()
 
 class User(db.Model):
-    id = db.Column(db.Integer, primary_key=True)
-    email = db.Column(db.String(120), unique=True, nullable=False)
-    password = db.Column(db.String(80), unique=False, nullable=False)
-    is_active = db.Column(db.Boolean(), unique=False, nullable=False)
-
-    def __repr__(self):
-        return '<User %r>' % self.username
-
-    def serialize(self):
-        return {
-            "id": self.id,
-            "email": self.email,
-            # do not serialize the password, its a security breach
-        }
\ No newline at end of file
+    __tablename__ = 'user'
+    id = db.Column(db.Integer, unique=True, primary_key=True)
+    name = db.Column(db.String)
+    mail = db.Column(db.String)
+    password = db.Column(db.String) 
+    age = db.Column(db.Integer, nullable=True)
+    localization = db.Column(db.String, nullable=True)
+    bio = db.Column(db.String, nullable=True)
+    
+    # #followers = db.relationship("Followers",
+    # secondary=association_table)
+
+class Trip(db.Model):
+    __tablename__ = 'trip'
+    id = db.Column(db.Integer, unique=True, primary_key=True)
+    country = db.Column(db.String)
+    cities = db.Column(db.String) 
+    activities = db.Column(db.String)
+    done = db.Column(db.Boolean, default=False) 
+
+class Share_Trip(db.Model):
+    __tablename__ = 'share_trip'
+    id = db.Column(db.Integer, unique=True, primary_key=True)
+    trip_id = db.Column(db.Integer, db.ForeignKey("trip.id"))
+    user_id = db.Column(db.Integer, db.ForeignKey("user.id")) 
```

Comment:
En esta mejor hagamos una association table https://docs.sqlalchemy.org/en/14/orm/basic_relationships.html#many-to-many